code/game/objects/structures/signs/signs_interactive.dm

```diff
@@ -24,3 +24,56 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/calendar, 32)
 		. += span_info("Events:")
 		for(var/holidayname in SSevents.holidays)
 			. += span_info("[holidayname]")
+
+GLOBAL_LIST_EMPTY(map_delamination_counters)
+
+/obj/structure/sign/delamination_counter
+	name = "delamination counter"
+	sign_change_name = "Flip Sign- Supermatter Delamination"
+	desc = "A pair of flip signs describe how long it's been since the last delamination incident."
+	icon_state = "days_since_explosion"
+	is_editable = TRUE
+	var/since_last = 0
+
+MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/delamination_counter, 32)
+
+/obj/structure/sign/delamination_counter/Initialize(mapload)
+	. = ..()
+	if (mapload)
+		GLOB.map_delamination_counters += src
+	else
+		update_count(SSpersistence.rounds_since_engine_exploded)
+
+/obj/structure/sign/delamination_counter/Destroy()
+	GLOB.map_delamination_counters -= src // Just in case.
+	return ..()
+
+/obj/structure/sign/delamination_counter/proc/update_count(new_count)
+	since_last = min(new_count, 99)
+	update_appearance()
+
+/obj/structure/sign/delamination_counter/update_overlays()
+	. = ..()
+
+	var/ones = since_last % 10
```

Comment:
Would it not be better to just use maptext here?

Reply:
After a bit of messing about, yes, but I think the 5px smallfonts characters are a bit uglier than the sprite overlays as they're a pixel thinner.
![image](https://user-images.githubusercontent.com/7483112/192600479-93b8763c-e2b3-4e2d-b71b-51c4cc38e9f3.png)

If adding 10 overlays to bloat a dmi file is more of a concern than the fact it looks a little worse, I can switch to maptext.